react-client/src/index.scss

```diff
@@ -91,3 +164,233 @@ main {
 .data-heading {
   font-weight: bold;
 }
+
+
+
+.dmpui-heading {
+  text-align: left;
+  margin-top: 1rem;
+
+  h1,
+  .h1 {
+    margin-bottom: 1.25rem;
+  }
+
+  h2,
+  .h2 {
+    margin-bottom: 1.25rem;
+  }
+
+  &.with-action-button {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+}
+
+.form-wrapper {
+  text-align: left;
+  padding: 20px;
+  border: 1px solid #eee;
+  margin-bottom: 2rem;
+  border-radius: 12px;
+}
+
+.plan-steps {
+  text-align: left;
+  padding: 2.5rem 2.5rem 2.5rem 2.5rem;
+  border: 1px solid #eee;
+  margin-bottom: 2rem;
+  border-radius: 12px;
+}
+
+.page-actions {
+  margin: 2rem 0;
+  display: flex;
+  justify-content: flex-end;
+
+  button {
+    margin-left: 10px;
+  }
+}
+
+
+.dmpdui-top-actions {
+  margin: 2rem 0 1rem 0;
+  display: flex;
+  justify-content: flex-end;
+
+  button {
+    margin-left: 10px;
+  }
+}
+
+.plan-steps-step {
+  display: flex;
+  border-bottom: 1px solid #eee;
+  padding: 1.125rem 0;
+  justify-content: space-between;
+  align-items: center;
+
+  &.last {
+    border-bottom: 1px solid transparent;
+  }
+
+  p {
+    line-height: normal;
+    color: var(--dmpui-brand-blue-500);
+    text-decoration: none;
+    margin: 0;
+    padding: 0;
+    font-size: 1.125rem;
+  }
+
+  a {
+    line-height: normal;
+    color: var(--dmpui-brand-blue-500);
+    text-decoration: none;
+    margin: 0;
+    padding: 0;
+
+
+    &:hover {
+      outline: none !important;
+      text-decoration: underline;
+    }
+
+    &:active,
+    &:focus {
+      color: #0b0c0c;
+      background-color: #fd0;
+      box-shadow: 0 -2px 0px 0px rgba(255, 221, 0, 0.281), 0 4px rgba(255, 221, 0, 0.281);
+      text-decoration: none;
+      -webkit-box-decoration-break: clone;
+      box-decoration-break: clone;
+      outline: none !important;
+      text-decoration: underline;
+    }
+  }
+
+}
+
+.step-status {
+  padding: 0.3rem 0.8rem;
+  background: var(--dmpui-brand-gray-100);
+  border-radius: 6px;
+  font-size: 0.8rem;
+  font-weight: 300;
+  line-height: normal;
+  color: var(--dmpui-brand-blue-900);
+
+  &.status-completed {
+    background-color: var(--dmpui-brand-green-100);
+    color: var(--dmpui-brand-blue-900);
+  }
+
+  &.status-recommended {
+    background-color: var(--dmpui-brand-blue-100);
+    color: var(--dmpui-brand-blue-900);
+  }
+
+  &.status-notstart {
+    background-color: var(--dmpui-brand-blue-200);
+    color: var(--dmpui-brand-blue-900);
+  }
+}
+
+
+.dmpui-search-form-container {
+  padding: 2rem;
+  background-color: var(--dmpui-brand-blue-100);
+}
+
+
+
+
+
+
+
+@import './components/responsive';
+
+#Dashboard {}
```

Comment:
What was the reason for moving the `#Dashboard` related styles into the `index.scss`?